src/main/java/org/embulk/filter/column/ColumnVisitorImpl.java

```diff
@@ -244,13 +244,13 @@ public void jsonColumn(Column outputColumn)
                 pageBuilder.setNull(outputColumn);
             }
             else {
-                String jsonPath = new StringBuilder("$.").append(outputColumn.getName()).toString();
+                String jsonPath = new StringBuilder("$['").append(outputColumn.getName()).append("']").toString();
```

Comment:
What happens if `outputColumn.getName()` has `'` (single quotes)? It looks like it will be broken.